README.md

```diff
@@ -54,7 +54,7 @@ docker export b38a367a8a05 > ppc64.tar
 
 The ubuntu version 14.04 is selected for better compatibility.
 
-- for `x86_64-musl` they are from https://musl.cc/. Last updated from build 23-Nov-2021 04:50 (7e2fa1cbc6b6c23d15e7b65015777e89  x86_64-linux-musl-native.tgz).
+- for `x86_64-musl` they are from https://musl.cc/. Last updated from build 23-Nov-2021 04:50 (7e2fa1cbc6b6c23d15e7b65015777e89  x86_64-linux-musl-native.tgz). Binaries `crt1.o`, `crti.o`, `libc.a` are rebuilt from https://git.musl-libc.org/ (commit dd1e63c3) configured with `--enable-debug`.
```

Comment:
Several issues:
* Unless it's really necessary I'd avoid using top of the HEAD as a reference, specially taking into account that the commit is about a revert. It's must better to use a stable release.
* Please update headers and binaries together, otherwise they don't match.
* I'm guessing that you haven't setup build flags, so you are likely building an unoptimized version of musl. Please include build flags. For example the ones used in the official binaries: https://conf.musl.cc/plain_20211122_11-2-1.txt